### DISCUSSION TITLE
Import and/or export the project data in csv

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16502] [Other Feature Feedback, Questions, & Ideas]
Hi all! Hope to contribute to GitHub

#### 1. Feature name
Import and/or export the project data in csv

#### 2. Feature description
I would like to import and export the project data in csv

#### 2.1 Ui/Ux
![image](https://user-images.githubusercontent.com/95204402/168226539-a2b36919-deaa-4721-be79-f5584d91be36.png)

#### 2.2 User stories
1. As a user I can export project data in csv for backup
2. As a user, I can import csv data that was from the backup I made inside the project

#### 2.3 Functional requirements
1. It is necessary to write a csv file with the information (information present in the columns) of the project. Note: The csv file writing mode refers to the project related csv data export requirement
2. It is necessary to read the csv file with the information (information present in the columns) of the project. Note: The csv file read mode refers to the requirement to import the csv data into the project

#### 3. Notes
1. I would like positive or negative feedback or criticism on whether or not this idea is a good idea.
2. This allows developers or team of developers to backup their projects.

**Comment by ghost** [comment]
This is cool.

**Comment by mulugetagessei** [comment]
any success story regarding the "**Export**/**Import**" projects to ***.csv** - so, that the exported ***.csv** file can be imported to another Org/Repo?
